test/parallel/test-runner-inspect.mjs

```diff
@@ -79,5 +79,5 @@ tmpdir.refresh();
   assert.strictEqual(code, 0);
   assert.strictEqual(signal, null);
   const files = await getCoveredFiles(coverageDirectory);
-  assert.ok(files.has(fixtures.fileURL('v8-coverage/basic.js').href));
+  assert.ok(files.has(fixtures.fileHref('v8-coverage', 'basic.js')));
```

Comment:
We can keep the forward slash. These are URLs, it’s always a forward slash.

Reply:
The thing is we pass that to `node:path` internally, then run `pathToFileURL` on top of this. I've learned since that it doesn't matter, because Windows still treat `/` as a path separator, so using one or multiple arguments still produces the same result.